lib/src/cli_util.dart

```diff
@@ -0,0 +1,267 @@
+// Copyright 2015 Google. All rights reserved. Use of this source code is
```

Comment:
Some of the methods in this file are not yet directly unit tested.  That's because they are mostly copied and pasted from unscripted with slight modifications, and so I want to factor them out into a separate library or package that can be shared by unscripted and grinder.  Plan to do that as part of implementing the annotation based API.